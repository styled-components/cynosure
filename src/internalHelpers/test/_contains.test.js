// @flow

import contains from '../_contains'

describe('contains', () => {
  it('should return `true` if the array contains the value', () => {
    expect(contains(1, [1, 2, 3])).toMatchSnapshot()
  })

  it('should return `false` if the array contains the value', () => {
    expect(contains(4, [1, 2, 3])).toMatchSnapshot()
  })
})
