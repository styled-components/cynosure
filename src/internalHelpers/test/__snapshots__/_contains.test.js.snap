exports[`contains should return \`false\` if the array contains the value 1`] = `false`;

exports[`contains should return \`true\` if the array contains the value 1`] = `true`;
